install/zowe-install.sh

```diff
@@ -148,6 +148,22 @@ cp $INSTALL_DIR/scripts/ocopyshr.sh $ZOWE_ROOT_DIR/scripts/internal/ocopyshr.sh
 cp $INSTALL_DIR/scripts/ocopyshr.clist $ZOWE_ROOT_DIR/scripts/internal/ocopyshr.clist
 echo "Copying the run-zowe.sh into "$ZOWE_ROOT_DIR/scripts/internal >> $LOG_FILE
 
+# Create the /fingerprint directory in the ZOWE_ROOT_DIR runtime directory,
+# if it exists in the INSTALL_DIR driectory
+if [[ -d $INSTALL_DIR/fingerprint ]]
+then
+  echo OK: Fingerprint exists in install directory and will be copied to runtime 
```

Comment:
Please remove the debug echos, or move them to the log

Reply:
Done.